heat_and_trees/heat_and_trees.ipynb

```diff
@@ -543,7 +572,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": false
+   },
    "outputs": [],
    "source": [
     "temp_f.hvplot(x='x', y='y', groupby='band', cmap='fire_r', \n",
```

Comment:
Hmm instead of calling `.layout()` on the plot I thought passing `subplots=True` would work. Something to discuss during our meeting.

Reply:
I think using `subplots=True` will create a widget that we can use to switch between the plots. It doesn't render both plots side by side the way `layout()` does.

Reply:
I got confused. `subplots=True` works when using `by` (not `groupby`) or a grid. I think calling `.layout()` is a fine workaround. It's for example mentioned here https://github.com/holoviz/hvplot/issues/157.